src/main/java/org/oscii/api/LexiconProtocol.java

```diff
@@ -139,6 +163,71 @@ private void addSynonyms(Request request, Response response) {
         response.synonyms = response.synonyms.stream().distinct().collect(toList());
     }
 
+    /**
+     * Adds the raw word vector for a query to the response.
+     *
+     * Example:
+     * http://localhost:8090/translate/lexicon?query=explain&wordvec=true
```

Comment:
If you change request `wordvec`->`embedding`, then this example needs to change too.